dot/network/gossip_integration_test.go

```diff
@@ -0,0 +1,111 @@
+// Copyright 2021 ChainSafe Systems (ON)
+// SPDX-License-Identifier: LGPL-3.0-only
+
+//go:build integration
+
+package network
+
+import (
+	"testing"
+	"time"
+
+	"github.com/ChainSafe/gossamer/dot/types"
+
+	"github.com/stretchr/testify/require"
+)
+
+// test gossip messages to connected peers
+func TestGossip(t *testing.T) {
+	if testing.Short() {
+		t.Skip("skipping TestGossip; currently, nothing is gossiped")
+	}
+
```

Comment:
Do we really want this? It's already tagged as integration, I think we don't need to check for `-short`